### DISCUSSION TITLE
"HighEnergyPhysics-9.0.zip"

### DISCUSSION THREAD

**Discussion by marti2216** [FeynCalc/feyncalc#107] [Q&A]
Hello,
How and where can I download and install "HighEnergyPhysics-9.0.zip" for FeynCalc? The FCGuide talks and requires this package but I cannot find it anywhere.
Thanks

**Comment by vsht** [comment]
FCGuide is very much outdated, so I wouldn't really use it as a reference. Rather go with

https://github.com/FeynCalc/feyncalc/wiki/Installation#st_automatic_installation
https://github.com/FeynCalc/feyncalc/wiki/Documentation#tutorials
https://github.com/FeynCalc/feyncalc/wiki/Documentation#online-reference
